wrapper.go

```diff
@@ -9,8 +9,9 @@ type writer struct {
 	Sep string
```

Comment:
This field is unused, and this type isn't exported. Can we remove it?

Reply:
I though so too. I will remove it.